etl/src/main/java/org/pih/hivmigration/etl/sql/TreatmentObsMigrator.groovy

```diff
@@ -0,0 +1,327 @@
+                    WHEN 'yes_continue' THEN concept_uuid_from_mapping('CIEL', '160037')
+                    WHEN 'yes_deferred' THEN concept_uuid_from_mapping('PIH', '7262')
+                    WHEN 'yes_immediately' THEN concept_uuid_from_mapping('PIH', '1619')
+                    WHEN 'yes_ptme' THEN concept_uuid_from_mapping('PIH', '1619')  -- yes, immediately
+                    WHEN 'yes_refused' THEN concept_uuid_from_mapping('CIEL', '127750')
+                    WHEN 'in_progress' THEN concept_uuid_from_mapping('CIEL', '160037')  -- yes, continue
+                END
+            FROM hivmigration_observations
+            WHERE observation = 'arv_treatment_needed'
+                AND value IS NOT NULL;  -- only 2 rows where it is null exist
+        ''')
+
+        // TODO: these values are all have source_encounter_type 'food_study'. Figure out what to do with them.
+//        executeMysql("Migrate ART Start date", '''
+//            INSERT INTO tmp_obs (source_encounter_id, concept_uuid, value_datetime)
+//            SELECT
+//                source_encounter_id,
+//                concept_uuid_from_mapping('CIEL', '159599'),
+//                value
+//            FROM hivmigration_observations
+//            WHERE observation = 'arv_start_date';
+//        ''')
+
+        executeMysql("CALL migrate_tmp_obs();")
+        executeMysql("CALL create_tmp_obs_table();")
+        setAutoIncrement("tmp_obs", "(SELECT max(obs_id) + 1 FROM obs)")
+
+        // TODO: map the common arv_regimen_other values into the new regimens
+        executeMysql("Set up for ARV Regimen migration", '''
+            CREATE TABLE hivmigration_arv_regimen (
+                obs_group_id int primary key auto_increment,
+                source_encounter_id int,
+                comments varchar(255)
+            );
+        ''')
+        setAutoIncrement('hivmigration_arv_regimen', '(select max(obs_id)+1 from obs)')
+
+        executeMysql("Migrate ARV Regimen", '''
+            INSERT INTO hivmigration_arv_regimen (source_encounter_id, comments)
+            SELECT source_encounter_id, comments
+            FROM hivmigration_ordered_other
+            WHERE ordered = 'arv_regimen';
+            
+            INSERT INTO tmp_obs (source_encounter_id, concept_uuid, obs_id)
+            SELECT source_encounter_id, concept_uuid_from_mapping('PIH', '10742'), obs_group_id
+            FROM hivmigration_arv_regimen;
+
+            INSERT INTO tmp_obs (source_encounter_id, concept_uuid, value_coded_uuid, obs_group_id)
+            SELECT
+                source_encounter_id,
+                concept_uuid_from_mapping('CIEL', '1282'),
+                CASE comments
+                    WHEN 'azt_3tc_efv' THEN concept_uuid_from_mapping('CIEL', '160124')
+                    WHEN 'azt_3tc_nvp' THEN concept_uuid_from_mapping('CIEL', '1652')
+                    ELSE concept_uuid_from_mapping('CIEL', '5424')  -- other
```

Comment:
This doesn't seem right.  We have lots more coded values.  We should map them, even if they are not on the current form.

Reply:
Ok, I added the other codes that exist in non-trivial quantities

![Screenshot from 2020-09-29 13-48-27](https://user-images.githubusercontent.com/1031876/94602478-c2dc0780-0249-11eb-9168-8cc178d14bef.png)